src/test_array_builder.cc

```diff
@@ -246,6 +258,21 @@ void createArray(Context ctx, std::string array_name, tiledb_array_type_t array_
           .add_dimension(Dimension::create<double>(ctx, "cols", {{1, 4}}, 4));
       break;
     }
+    case TILEDB_CHAR:{
+      domain.add_dimension(Dimension::create<char>(ctx, "rows", {{1, 4}}, 4))
+              .add_dimension(Dimension::create<char>(ctx, "cols", {{1, 4}}, 4));
+      break;
+    }
+    case TILEDB_STRING_ASCII:{
+    /*case TILEDB_STRING_UTF8:
+    case TILEDB_STRING_UTF16:
+    case TILEDB_STRING_UTF32:
+    case TILEDB_STRING_UCS2:
+    case TILEDB_STRING_UCS4:{*/
+      domain.add_dimension(Dimension::create(ctx, "rows",TILEDB_STRING_ASCII ,nullptr, nullptr))
```

Comment:
Please use `clang-format` to format this source file. Our core repository's CI enforces this styling, but we don't have CI on this repo.